.mocharc.yml

```diff
@@ -1,6 +1,6 @@
 require:
   - ts-node/register
   - test/rootMochaHooks.ts
-reporter: tap # more flat, also the only one that mentions totals for found, skipped and passed
+# reporter: tap # more flat, also the only one that mentions totals for found, skipped and passed
```

Comment:
Is this comment required?

Reply:
I would keep that commented. Because we might have to change to this setting if the log outputs do not look good.